wazimap_ng/points/tasks.py

```diff
@@ -29,19 +20,30 @@ def process_uploaded_file(point_file, subtheme, **kwargs):
     Read files using pandas according to file extension.
     After reading data convert to list rather than using numpy array.
     """
-    filename = point_file.document.name
+    buffer = point_file.document.open("rb")
```

Comment:
this code is now duplicated here and in [dataset upload](https://github.com/OpenUpSA/wazimap-ng/blob/staging/wazimap_ng/datasets/tasks/process_uploaded_file.py#L32) let's consolidate in utils as well.

Reply:
I created a new function for that: https://github.com/OpenUpSA/wazimap-ng/pull/169/commits/2b66c86c09b6410d9177538871c8f6465afd3519